### PR TITLE
Support for representing and evaluating Чебышёв series

### PR DESCRIPTION
**Pull request by pleroy**

### CODE REVIEW

File: numerics/чебышёв_series.hpp [hunk 1]
```diff
@@ -0,0 +1,41 @@
+﻿#pragma once
+
+#include <vector>
+
+#include "geometry/named_quantities.hpp"
+#include "quantities/quantities.hpp"
+
+// Spelling: Чебышёв ЧЕБЫШЁВ чебышёв
```
Comment (eggrobin) [inline numerics/чебышёв_series.hpp]: Remove

File: numerics/чебышёв_series_body.hpp [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  // Precomputed to save operations at the expense of some accuracy loss.
+  Time const duration = t_max_ - t_min_;
+  t_mean_ = t_min_ + 0.5 * duration;
+  two_over_duration_ = 2 / duration;
+}
+
+template<typename Scalar>
+Scalar ЧебышёвSeries<Scalar>::Evaluate(Instant const& t) const {
+  double const scaled_t = (t - t_mean_) * two_over_duration_;
+  double const two_scaled_t = scaled_t + scaled_t;
```
Comment (eggrobin) [inline numerics/чебышёв_series_body.hpp]: Does the compiler not optimize this on its own? Is the performance impact measurable?

File: numerics/чебышёв_series.hpp (hunk 1 above)
Comment (pleroy) [inline numerics/чебышёв_series.hpp]: No, please, let's keep it here, it's a pain to have to look it up and do the case conversion.

File: numerics/чебышёв_series_body.hpp (hunk 2 above)
Comment (pleroy) [inline numerics/чебышёв_series_body.hpp]: I am pretty sure that any decent compiler will hoist this out of the loop, but why depend on that?

**Pull request merged**